packages/desktop-client/src/components/modals/CreateLocalAccountModal.tsx

```diff
@@ -32,9 +33,14 @@ export function CreateLocalAccountModal({
   const [offbudget, setOffbudget] = useState(false);
   const [balance, setBalance] = useState('0');
 
-  const [nameError, setNameError] = useState(false);
+  const [nameError, setNameError] = useState({
+    error: false,
+    message: 'Name is required',
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏initially there should be no error
```suggestion
    message: '',
```